S/SCIP/build_tarballs.jl

```diff
@@ -17,9 +18,15 @@ script = raw"""
 # remove when CMake accounts for this
 if [[ "${target}" == *86*-linux-gnu ]]; then
    export LDFLAGS="-lrt"
+elif [[ "${target}" == *-mingw* ]]; then
```

Comment:
was this tested on windows? So far I've never managed to have the windows binaries work